web/src/app/icons/page.tsx

```diff
@@ -1,70 +1,83 @@
-import { BASE_URL } from "@/constants"
+import { BASE_URL, BROWSE_KEYWORDS, DEFAULT_OG_IMAGE, GITHUB_URL, ORGANIZATION_NAME, ORGANIZATION_SCHEMA, SITE_NAME, SITE_TAGLINE, TITLE_SEPARATOR, WEB_URL, getBrowseDescription } from "@/constants"
 import { getIconsArray } from "@/lib/api"
 import type { Metadata } from "next"
 import { IconSearch } from "./components/icon-search"
+import Script from "next/script"
 
 export async function generateMetadata(): Promise<Metadata> {
 	const icons = await getIconsArray()
 	const totalIcons = icons.length
 
+	const title = `Browse Icons ${TITLE_SEPARATOR} ${SITE_NAME}`
+	const description = getBrowseDescription(totalIcons)
+
 	return {
-		title: "Browse Icons | Free Dashboard Icons",
-		description: `Search and browse through our collection of ${totalIcons} curated icons for services, applications and tools, designed specifically for dashboards and app directories.`,
-		keywords: [
-			"browse icons",
-			"dashboard icons",
-			"icon search",
-			"service icons",
-			"application icons",
-			"tool icons",
-			"web dashboard",
-			"app directory",
-		],
+		title,
+		description,
+		keywords: BROWSE_KEYWORDS,
 		openGraph: {
-			title: "Browse Icons | Free Dashboard Icons",
-			description: `Search and browse through our collection of ${totalIcons} curated icons for services, applications and tools, designed specifically for dashboards and app directories.`,
+			title: `Browse Icons ${TITLE_SEPARATOR} ${SITE_NAME} ${TITLE_SEPARATOR} ${SITE_TAGLINE}`,
```

Comment:
Wtf is this????? Don't use constants for titles

Reply:
They are used everywhere throughout the site on multiple pages, how do you want to do it whilst keeping it dynamic? This seems like the easiest solution.

Reply:
They're used everywhere throughout the site and should be dynamic. Why wouldn't you?